docs/source/user-guide/supported_models.rst

```diff
@@ -0,0 +1,56 @@
+.. _supported_models:
+
+====================
+Supported Models
+====================
+
+Deepchecks supports a minmal version of the sklearn api.
+
+
+Required API
+=============
+
+Regression
+----------
+
+All that is necessary for for regression models is the ``predict`` function.
+
+Example of an simple model:
+
+>>> class simple_regression_model:
+...     def predict(X: pd.DataFrame) -> pd.Series:
+...         ...
+
+Classification
+--------------
+
+For classification models, we require also the ``predict_proba`` function:
+
+>>> class simple_classification_model:
+...     def predict(X: pd.DataFrame) -> pd.Series:
+...         ...
+...     def predict_proba(X: pd.DataFrame) -> pd.Series:
+...         ...
+
+
+Other Helpful APIs
+==================
+
+Feature importance
+-------------------
+
+Deepchecks can calculate feature importance using ``permutational_importance``.
```

Comment:
I think some elaborations on permutational importance are missing. Maybe we can send them to sklearn docs, or document here.

Reply:
it's also confusing that it looks like a "term". if it's a link to a specific function then you can link here to it's API reference. otherwise it will look like something that people want (and try to) click but it leads them no where and remains unclear